gin/config_parser.py

```diff
@@ -437,6 +452,42 @@ def _maybe_parse_configurable_reference(self):
 
     return True, reference
 
+  def _maybe_parse_identifier_reference(self):
+    """Try to parse a identifier reference([package.package.]var_or_fn_or_cls_name[()])"""
```

Comment:
Is '()' allowed? The function doesn't seemed to handle `()`

Reply:
my bad, evaluate with `*args` and `*kwargs* is allowed for ordinary reference, it's not  included in this pr